discord/user.go

```diff
@@ -129,31 +129,37 @@ const (
 )
 
 type Activity struct {
```

Comment:
Since comments for user only fields are already being added, the remaining fields may as well be documented.

```suggestion
// https://discord.com/developers/docs/topics/gateway#activity-object-activity-structure
type Activity struct {
```